python/simpler/remote_l3_session.py

```
@@ -585,31 +584,40 @@ def _materialize_task_args(
+    returns.
     """
-    if len(args.remote_desc) != len(args.tensor_metadata):
-        raise ValueError("remote TASK descriptor count does not match tensor metadata count")
+    if len(args.remote_desc) != len(args.tensors):
+        raise ValueError("remote TASK descriptor count does not match tensor count")
     task_args = TaskArgs()
     inline_backings: list[Buffer] = []
 
     try:
-        for tensor, sidecar in zip(args.tensor_metadata, args.remote_desc):
+        for tensor, sidecar in zip(args.tensors, args.remote_desc):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)